### PR TITLE
Add JankStats as a demo to ComposeAdvanced

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/util/JankPrinter.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    nonJank++
+                }
+            }.apply {
+                jankHeuristicMultiplier = 3f
+            }
+        }
+    }
+
+    fun setRouteState(route: String?) {
+        if (route != null) {
```
Comment (shumelchyk) [inline ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/util/JankPrinter.kt:71]: Nit:
```
    stateHolder?.state?.let {
        if (route != null) it.addState("route", route) else it.removeState("route")
    }
```

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/util/JankPrinter.kt [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                    val route =
+                        it.states.find { state -> state.stateName == "route" }?.state.orEmpty()
+                    val duration = it.frameDurationUiNanos.nanosToMillis()
+                    Log.w("Jank", "Jank $duration route:$route non:$nonJank")
+                    nonJank = 0
+                } else {
+                    nonJank++
+                }
+            }.apply {
+                jankHeuristicMultiplier = 3f
```
Comment (shumelchyk) [inline ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/util/JankPrinter.kt:65]: Worth a comment why this value is used and not the default one (2f)

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/WearApp.kt [hunk 3]
```diff
@@ -0,0 +1,268 @@
+package com.example.android.wearable.composeadvanced.presentation
```
[the remark this answers is not in the document]
Comment (shumelchyk) [reply ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/WearApp.kt:16]: Already resolved by cleanup commit.

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/util/JankPrinter.kt (hunk 1 above)
Comment (shumelchyk) [reply ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/util/JankPrinter.kt]: Feel free to skip then (I didn't check in the editor)

**Comment by yschimke** [comment]
@BowerSteve landed, I'll follow up with any review fixes.

**Pull request merged**